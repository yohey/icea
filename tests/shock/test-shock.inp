#
# Example 7 mod
#
problem case = Example-7-mod
        shock
        p,mmhg = 10, 20
        u1 = 1000, 1100, 1200, 1250, 1300, 1350, 1400
        incd
        froz
        eql

reac    name = H2  moles = 0.050  t(k) 300.00
        name = O2  moles = 0.050  t(k) 300.00
        name = Ar  moles = 0.900  t(k) 300.00

end
