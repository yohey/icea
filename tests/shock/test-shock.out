
 *******************************************************************************

         NASA-GLENN CHEMICAL EQUILIBRIUM PROGRAM CEA2, MAY 21, 2004
                   BY  BONNIE MCBRIDE AND SANFORD GORDON
      REFS: NASA RP-1311, PART I, 1994 AND NASA RP-1311, PART II, 1996

 *******************************************************************************



 #
 # Example 7 mod
 #
 problem case = Example-7-mod
         shock
         p,mmhg = 10, 20
         u1 = 1000, 1100, 1200, 1250, 1300, 1350, 1400
         incd
         froz
         eql
  
 reac    name = H2  moles = 0.050  t(k) 300.00
         name = O2  moles = 0.050  t(k) 300.00
         name = Ar  moles = 0.900  t(k) 300.00
  
 end

 OPTIONS: TP=F  HP=F  SP=F  TV=F  UV=F  SV=F  DETN=F  SHOCK=T  REFL=F  INCD=T
 RKT=F  FROZ=T  EQL=T  IONS=F  SIUNIT=T  DEBUGF=F  SHKDBG=F  DETDBG=F  TRNSPT=F

 TRACE= 0.00E+00  S/R= 0.000000E+00  H/R= 0.000000E+00  U/R= 0.000000E+00

 P,BAR =     0.013332     0.026664

    REACTANT           MOLES    (ENERGY/R),K   TEMP,K  DENSITY
        EXPLODED FORMULA
 N: H2               0.050000   0.641759E+01   300.00  0.0000
          H  2.00000
 N: O2               0.050000   0.653777E+01   300.00  0.0000
          O  2.00000
 N: Ar               0.900000   0.462500E+01   300.00  0.0000
          AR 1.00000

  SPECIES BEING CONSIDERED IN THIS SYSTEM
 (CONDENSED PHASE MAY HAVE NAME LISTED SEVERAL TIMES)
  LAST thermo.inp UPDATE:    9/09/04

  g 3/98  *Ar              g 6/97  *H               g 4/02  HO2            
  tpis78  *H2              g 8/89  H2O              g 6/99  H2O2           
  g 5/97  *O               g 4/02  *OH              tpis89  *O2            
  g 8/01  O3               g11/99  H2O(cr)          g 8/01  H2O(L)         
  g 8/01  H2O(L)         

   *** INPUT FOR SHOCK PROBLEMS ***

 INCDEQ = T   REFLEQ = F   INCDFZ = T    REFLFZ = F

 U1 =    1.000000E+03 1.100000E+03 1.200000E+03 1.250000E+03 1.300000E+03
         1.350000E+03 1.400000E+03

 MACH1 = 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00 0.000000E+00
         0.000000E+00 0.000000E+00

 O/F =   0.000000

                       EFFECTIVE FUEL     EFFECTIVE OXIDANT        MIXTURE
 ENTHALPY                  h(2)/R              h(1)/R               h0/R
 (KG-MOL)(K)/KG        0.12774942E+00      0.00000000E+00      0.12774942E+00

 KG-FORM.WT./KG             bi(2)               bi(1)               b0i
  *H                   0.26557650E-02      0.00000000E+00      0.26557650E-02
  *O                   0.26557650E-02      0.00000000E+00      0.26557650E-02
  *Ar                  0.23901885E-01      0.00000000E+00      0.23901885E-01




                         SHOCK WAVE PARAMETERS ASSUMING

                 EQUILIBRIUM COMPOSITION FOR INCIDENT SHOCKED CONDITIONS


 CASE = Example-7-mod  

             REACTANT                       MOLES         ENERGY      TEMP
                                                         KJ/KG-MOL      K
 NAME        H2                           0.0500000        53.359    300.000
 NAME        O2                           0.0500000        54.358    300.000
 NAME        Ar                           0.9000000        38.455    300.000

 O/F=    0.00000  %FUEL=  0.000000  R,EQ.RATIO= 0.500000  PHI,EQ.RATIO= 0.000000

 INITIAL GAS (1)
 MACH NUMBER1      3.0480   3.3528   3.6576   3.8100   3.9624   4.1148   4.2672
 U1, M/SEC        1000.00  1100.00  1200.00  1250.00  1300.00  1350.00  1400.00
 P, BAR           0.01333  0.02666  0.02666  0.02666  0.02666  0.02666  0.02666
 T, K              300.00   300.00   300.00   300.00   300.00   300.00   300.00
 RHO, KG/CU M    2.0126-2 4.0252-2 4.0252-2 4.0252-2 4.0252-2 4.0252-2 4.0252-2
 H, KJ/KG          1.0622   1.0622   1.0622   1.0622   1.0622   1.0622   1.0622
 U, KJ/KG         -65.182  -65.182  -65.182  -65.182  -65.182  -65.182  -65.182
 G, KJ/KG        -1556.26 -1510.35 -1510.35 -1510.35 -1510.35 -1510.35 -1510.35
 S, KJ/(KG)(K)     5.1911   5.0380   5.0380   5.0380   5.0380   5.0380   5.0380

 M, (1/n)          37.654   37.654   37.654   37.654   37.654   37.654   37.654
 Cp, KJ/(KG)(K)    0.5742   0.5742   0.5742   0.5742   0.5742   0.5742   0.5742
 GAMMAs            1.6249   1.6249   1.6249   1.6249   1.6249   1.6249   1.6249
 SON VEL,M/SEC      328.1    328.1    328.1    328.1    328.1    328.1    328.1

       WARNING!!  NO CONVERGENCE FOR u1=  1000.0
  ANSWERS NOT RELIABLE, SOLUTION MAY NOT EXIST (SHCK)

 SHOCKED GAS (2)--INCIDENT--EQUILIBRIUM
 U2, M/SEC         703.56   666.71   575.84   559.89   548.57   539.62   531.73
 P, BAR           0.08449  0.21851  0.32815  0.37389  0.41987  0.46702  0.51596
 T, K             1371.90  1528.26  1816.81  1931.88  2043.24  2151.82  2257.19
 RHO, KG/CU M    2.8606-2 6.6411-2 8.3881-2 8.9866-2 9.5388-2 1.0070-1 1.0598-1
 H, KJ/KG          292.40   383.81   555.27   625.58   695.60   766.72   839.69
 U, KJ/KG         -2.9591   54.783   164.06   209.52   255.43   302.94   352.85
 G, KJ/KG        -7331.25 -7892.51 -9311.30 -9884.07 -10440.8 -10985.1 -11513.8
 S, KJ/(KG)(K)     5.5570   5.4155   5.4307   5.4401   5.4504   5.4613   5.4730

 M, (1/n)          38.619   38.619   38.613   38.607   38.596   38.577   38.549
 (dLV/dLP)t      -1.00000 -1.00001 -1.00005 -1.00011 -1.00022 -1.00042 -1.00074
 (dLV/dLT)p        1.0001   1.0003   1.0020   1.0039   1.0073   1.0127   1.0211
 Cp, KJ/(KG)(K)    0.5828   0.5872   0.6053   0.6205   0.6435   0.6775   0.7255
 GAMMAs            1.5859   1.5794   1.5555   1.5378   1.5139   1.4834   1.4477
 SON VEL,M/SEC      684.4    720.9    780.1    799.9    816.3    829.4    839.5

 P2/P1              6.654    8.195   12.307   14.022   15.746   17.515   19.350
 T2/T1              4.656    5.094    6.056    6.440    6.811    7.173    7.524
 M2/M1             1.0256   1.0256   1.0255   1.0253   1.0250   1.0245   1.0238
 RHO2/RHO1         1.4213   1.6499   2.0839   2.2326   2.3698   2.5018   2.6329
 V2, M/SEC         296.44   433.29   624.16   690.11   751.43   810.38   868.27

 MOLE FRACTIONS

 *Ar             9.2307-1 9.2306-1 9.2293-1 9.2277-1 9.2251-1 9.2207-1 9.2140-1
 *H              6.3883-9 7.2053-8 4.2600-6 1.5451-5 4.6715-5 1.2250-4 2.8387-4
 HO2             6.0024-9 2.5053-8 1.4565-7 2.5320-7 4.0648-7 6.1294-7 8.7509-7
 *H2             3.2025-7 1.8749-6 3.5013-5 8.8014-5 1.9380-4 3.8434-4 6.9522-4
 H2O             5.1271-2 5.1243-2 5.0993-2 5.0727-2 5.0296-2 4.9634-2 4.8672-2
 H2O2            2.225-10 9.438-10 4.488 -9 7.331 -9 1.116 -8 1.606 -8 2.197 -8
 *O              3.3353-7 2.0240-6 3.9706-5 1.0147-4 2.2697-4 4.5761-4 8.4353-4
 *OH             2.1761-5 7.2536-5 4.8708-4 8.8525-4 1.4747-3 2.2935-3 3.3581-3
 *O2             2.5635-2 2.5622-2 2.5514-2 2.5408-2 2.5251-2 2.5033-2 2.4751-2

  * THERMODYNAMIC PROPERTIES FITTED TO 20000.K

    PRODUCTS WHICH WERE CONSIDERED BUT WHOSE MOLE FRACTIONS
    WERE LESS THAN 5.000000E-09 FOR ALL ASSIGNED CONDITIONS

 O3              H2O(cr)         H2O(L)         




                         SHOCK WAVE PARAMETERS ASSUMING

                  FROZEN COMPOSITION FOR INCIDENT SHOCKED CONDITI1ONS


 CASE = Example-7-mod  

             REACTANT                       MOLES         ENERGY      TEMP
                                                         KJ/KG-MOL      K
 NAME        H2                           0.0500000        53.359    300.000
 NAME        O2                           0.0500000        54.358    300.000
 NAME        Ar                           0.9000000        38.455    300.000

 O/F=    0.00000  %FUEL=  0.000000  R,EQ.RATIO= 0.500000  PHI,EQ.RATIO= 0.000000

 INITIAL GAS (1)
 MACH NUMBER1      3.0480   3.3528   3.6576   3.8100   3.9624   4.1148   4.2672
 U1, M/SEC        1000.00  1100.00  1200.00  1250.00  1300.00  1350.00  1400.00
 P, BAR           0.01333  0.02666  0.02666  0.02666  0.02666  0.02666  0.02666
 T, K              300.00   300.00   300.00   300.00   300.00   300.00   300.00
 RHO, KG/CU M    2.0126-2 4.0252-2 4.0252-2 4.0252-2 4.0252-2 4.0252-2 4.0252-2
 H, KJ/KG          1.0622   1.0622   1.0622   1.0622   1.0622   1.0622   1.0622
 U, KJ/KG         -65.182  -65.182  -65.182  -65.182  -65.182  -65.182  -65.182
 G, KJ/KG        -1556.26 -1510.35 -1510.35 -1510.35 -1510.35 -1510.35 -1510.35
 S, KJ/(KG)(K)     5.1911   5.0380   5.0380   5.0380   5.0380   5.0380   5.0380

 M, (1/n)          37.654   37.654   37.654   37.654   37.654   37.654   37.654
 Cp, KJ/(KG)(K)    0.5742   0.5742   0.5742   0.5742   0.5742   0.5742   0.5742
 GAMMAs            1.6249   1.6249   1.6249   1.6249   1.6249   1.6249   1.6249
 SON VEL,M/SEC      328.1    328.1    328.1    328.1    328.1    328.1    328.1

 SHOCKED GAS (2)--INCIDENT--FROZEN
 U2, M/SEC         317.26   332.77   349.41   358.06   366.89   375.87   384.99
 P, BAR           0.15074  0.36638  0.43752  0.47544  0.51494  0.55601  0.59865
 T, K             1076.14  1247.03  1433.31  1532.25  1635.05  1741.69  1852.17
 RHO, KG/CU M    6.3438-2 1.3305-1 1.3824-1 1.4052-1 1.4263-1 1.4457-1 1.4637-1
 H, KJ/KG          450.75   550.72   660.02   718.21   778.76   841.67   906.95
 U, KJ/KG          213.13   275.36   343.52   379.87   417.72   457.08   497.97
 G, KJ/KG        -5354.24 -6039.09 -6975.05 -7475.96 -7998.88 -8543.88 -9111.04
 S, KJ/(KG)(K)     5.3943   5.2844   5.3269   5.3478   5.3684   5.3888   5.4088

 M, (1/n)          37.654   37.654   37.654   37.654   37.654   37.654   37.654
 Cp, KJ/(KG)(K)    0.5841   0.5858   0.5876   0.5886   0.5895   0.5904   0.5913
 GAMMAs            1.6078   1.6049   1.6019   1.6005   1.5989   1.5974   1.5960
 SON VEL,M/SEC      618.1    664.8    712.0    735.9    759.8    783.8    807.9

 P2/P1             11.307   13.740   16.408   17.831   19.312   20.852   22.451
 T2/T1              3.587    4.157    4.778    5.108    5.450    5.806    6.174
 M2/M1             1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
 RHO2/RHO1         3.1520   3.3056   3.4344   3.4911   3.5433   3.5917   3.6365
 V2, M/SEC         682.74   767.23   850.59   891.94   933.11   974.13  1015.01

 MOLE FRACTIONS

  *H2             0.05000  0.05000  0.05000  0.05000  0.05000  0.05000  0.05000
  *O2             0.05000  0.05000  0.05000  0.05000  0.05000  0.05000  0.05000
  *Ar             0.90000  0.90000  0.90000  0.90000  0.90000  0.90000  0.90000



